### PR TITLE
Template is up to date

### LINKED ISSUE #8: Add note to readme about most recent graduate
It might be helpful for a OSU graduate student interested in this template to know when it was most recently utilized.

### PR DESCRIPTION
**Pull request by j-c-cook**
Employees of the grad college nodded in unison with glazed eyes repeating, "If you're using Latex, you need to use the online template." 

closes #8

### CODE REVIEW

**Comment by j-c-cook** [comment]
I'll make my "up to date" PRs (#5 and #7) _ready for review_ once the grad college informs me that there are no more formatting issues.

**Comment by j-c-cook** [comment]
This PR is not ready for review yet.

**Comment by j-c-cook** [comment]
This PR is ready for review.

File: README.md [hunk 1]
```diff
@@ -6,5 +6,13 @@ This is a template for writing a thesis or dissertation using LaTeX at Oklahoma
 # Contributing
 Spot an error, bug, or want to contribute? Create a [branch](https://git-scm.com/book/en/v2/Git-Branching-Branches-in-a-Nutshell) and submit a [pull request.](https://github.com/mitchute/OSULaTeXTheisTemplate/compare?expand=1)
 
+# Usage
+This template has been successfully utilized by many OSU graduate students. The graduate college does provide a template of their own. However, any template could be used. The information in the following table may be useful for a student who is considering whether to utilize the graduate colleges template or this one. The table includes a non-exhaustive description of students who have utilized this template.
```
Comment (mitchute) [inline README.md:10]: > The graduate college does provide a template of their own.

They have a LaTeX template of their own? They had taken that down when I was there.

Comment (j-c-cook) [reply README.md:10]: In light of recent events (most likely your @mitchute experiences with them), the grad college did pay a couple of guys to come in and edit the existing template. 
```
Original author: Brian O'Dell (1999), revised by Ryan T. Ratliff (2005).
Current version by Paul Fili, Edward Richmond (2020).
```
The grad colleges template does not have the organization of this template, nor does it have hyper-referencing for figures and the TOC/LOF/LOT. I believe this template is more user friendly.

**Comment by j-c-cook** [comment]
It's up to you on whether or not this PR is merged. My reasoning for doing this:

Students will know how "up to date" this template is. I was warned that this template may not be up to date because you graduated two years ago. The approval of my thesis proves that this template (with the edits I have made #5, #7 and #11) is up to date and can be used.

**Comment by mitchute** [comment]
Yeah, this is fine with me. Merging.

**Pull request merged**